internal/connection_repository/sql_connection_locator_v2.go

```diff
@@ -217,3 +218,72 @@ func NewSqlGetConnectionsByOrgID(cfg *config.Config, database *sql.DB) (GetConne
 		return connectionsPerAccount, totalConnections, nil
 	}, nil
 }
+
+func NewGetAllConnections(cfg *config.Config, database *sql.DB, proxyFactory controller.ConnectorClientProxyFactory) (GetAllConnections, error) {
```

Comment:
Let's remove the proxy factory here and change the inner function to return `map[AccountID]map[ClientID]domain.ConnectorClientState`.